networking_oneview/ml2/drivers/oneview/neutron_oneview_client.py

```diff
@@ -209,6 +215,12 @@ def _create_network_on_oneview(self, name, network_type, seg_id):
         }
         return self.oneview_client.ethernet_networks.create(options)
 
+    def _verify_network_existence(self, name, type, seg_id):
+        comparator = lambda network: (network.name == name and
+                                      network.ethernetNetworkType == type and
+                                      network.vlanId == seg_id)
+        return self.oneview_client.ethernet_networks.get_all(filter=comparator)
```

Comment:
What is it supposed to happen when there is a Network with the same name, but all the other information is different? This will return no Network, therefore, the `_create_network_on_oneview` method will try to create a Network with a name that already exists on OneView. This will throw an Exception.

We need to decide if:

- We want to stop the creation if there is a Network with the same name, for it is not possible to create it, and then LOG that this happened.
- Or trust that if the name is the same, all the info is the same. In this case using `self.oneview_client.ethernet_networks.get_by('name', name)` is enough.